dashboard/app/models/script_level.rb

```diff
@@ -16,7 +15,6 @@
 #
 # Indexes
 #
-#  index_script_levels_on_level_id   (level_id)
```

Comment:
Do we intend to also resolve the TODO(ram) below (the four convenience methods) at some point?